using_images/db_images/mongodb.adoc

```diff
@@ -549,22 +571,7 @@ using `rs.initiate(config)`. The script waits until MongoDB elects the *PRIMARY*
 member of the cluster.
 
 Once the *PRIMARY* member has been elected, the entrypoint script starts
```

Comment:
I think entrypoint script is a sufficiently generic description of what it's happening here.  it's the script (including embedded scripts) that are executed for the entrypoint.  We're not trying to do a line by line code review here, just give a basic overview of how the image bootstraps.  As you said, anything more detailed is going to rot.